.github/workflows/ci_build.yaml

```diff
@@ -20,7 +20,14 @@ on:
       - "Dockerfile_amd"
       - "Dockerfile_intel"
     branches:
-      - 'main'
+      - "main"
+  workflow_dispatch:
+    inputs:
+      release-tests:
+        description: "Run release integration tests"
+        required: false
+        default: false
+        type: boolean
```

Comment:
I think you shoudl send that as an input of `build.yaml` too.

Reply:
Thanks, fixed!